### ISSUE TITLE
Sharing post by different users

### ISSUE THREAD

**Issue by karlkunc** [PHPfox-Official/phpfox-v4-issues#3005]
## What's happened?
When someone posts to the feed and a user shares that post it works fine. then a different user shares the shared post it will be seen in the feed, but if a different user comes along and shares it, the original post will not show up in the feed. It will just show that 2 previous people shared something.

## Steps to reproduce:
1. Did exactly what I wrote above
2. 
3. 

## What's expected?
... To show the original post no matter how many levels of sharing happened. 

## Browsers and Devices tested
mobile and browser
...

## Server information
latest version updates. 

phpfox_version: | 4.8.3
PHP Version: | 7.3.15
PHP Sapi: | fpm-fcgi
MySQL 5.5.5-10.1.32-MariaDB

...

## phpFox version
4.8.3

...

## Screenshots
...
![Screenshot_20210322-120515_Chrome](https://user-images.githubusercontent.com/74743076/112020786-0764a380-8b07-11eb-8b0c-e695fd5d1e14.jpg)

**Comment by harrison05** [comment]
Hi @karlkunc 

We can't reproduce this issue. Please submit a ticket in the client area if it still appears on your site, then we can check on your site directly.

Regards.

**Comment by karlkunc** [comment]
My apologies it only happens with the mobile apps that were just updated. The web version doesn't have the issue.

**Comment by harrison05** [comment]
Yes, we can reproduce it on Mobile App. We will check and fix it.

Thanks for your report.
